src/autotester/server/utils/tests/file_management_test.py

```diff
@@ -0,0 +1,111 @@
+from src.autotester.server.utils.file_management import *
+import os.path
+import re
+import tempfile
+import shutil
+from src.autotester.config import config
+FILES_DIRNAME = config["_workspace_contents", "_files_dir"]
+
+def test_clean_dir_name():
+    a = "markus/address"
+    b = a.replace("/", "_")
+    assert clean_dir_name(a) == b
+
+
+def test_random_tmpfile_name():
+    tmp_file = random_tmpfile_name()
+    dir_name = os.path.dirname(tmp_file)
+    base_name = os.path.basename(tmp_file)
+    assert os.path.exists(dir_name) and re.match(
+        r"[+\-]" "0" "[xX]" "0|" "([1-9A-Fa-f][1-9A-Fa-f]*)", base_name
+    )
+
+
+def test_recursive_iglob():
+    root_dir = tempfile.gettempdir()
```

Comment:
this will always give you an empty directory so your test will only ever check if this function works on an empty directory. Is that what you intend for this test?